runtime/moonbase/src/lib.rs

```diff
@@ -709,6 +721,7 @@ impl cumulus_pallet_parachain_system::Config for Runtime {
 	type XcmpMessageHandler = XcmpQueue;
 	type ReservedXcmpWeight = ReservedXcmpWeight;
 	type CheckAssociatedRelayNumber = cumulus_pallet_parachain_system::RelayNumberStrictlyIncreases;
```

Comment:
If we are setting `AllowMultipleBlocksPerSlot` to `true`, shouldn't we use `RelayNumberMonotonicallyIncreases` type here? Otherwise we might encounter inconsistencies while checking the number of para-blocks produced by relay-block.

Reply:
good catch